zipline/finance/controls.py

```diff
@@ -99,6 +99,35 @@ def validate(self,
         self.orders_placed += 1
 
 
+class RestrictedListOrder(TradingControl):
+    """
+    TradingControl representing a restricted list of securities that
+    cannot be ordered by the algorithm.
+    """
+
+    def __init__(self, restricted_list):
+        """
+        restricted list can be an iterable, or
+        an object that implements __contains__ for dynamic
```

Comment:
restricted_list can be an iterable or container for dynamic restrictions.

I think the convention is to say that something that implements `__contains__` is a container; however, this is not a big deal if you think this is more clear.